opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -392,14 +398,18 @@ def _convert_aggregation_temporality(
                     current_point.bucket_counts, previous_point.bucket_counts
                 )
             ]
+        max_ = max(current_point.max, previous_point.max)
+        min_ = min(current_point.min, previous_point.min)
```

Comment:
These can's be correct for cumulative -> delta conversion, right?

Reply:
Yeah great question. I'll bring up this point in the sig because as of now it doesn't look like we have enough information to accurately determine the min/max for that use case.

Reply:
Pasting this diagram for better explanation:

time --->

previous_point_delta:
          |-------|

previous_point_cumulative:
|-----------------|

current_point_cumulative:
|-------------------------|

current_point_delta:
                  |-------|

Reply:
I think the conversion of a Histogram current_point from cumulative to delta will never happen:

The `current_point` is obtained [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_view_instrument_match.py#L90). The aggregation temporality of a `Histogram` point is harcoded to [DELTA](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py#L280).

So, `current_point` will always have an aggregation temporality of DELTA.

Reply:
@srikanthccv 
Thoughts?

Reply:
Hmm, I would like this function to not make any assumptions and just take the input and return the output. If we don't want to support certain case just raise an exception or do something. What does spec recommend for histogram cum -> delta conversion?

Reply:
[Here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#histogram) it says this:

> The aggregation temporality also has implications on the min and max fields. Min and max are more useful for Delta temporality, since the values represented by Cumulative min and max will stabilize as more events are recorded. Additionally, it is possible to convert min and max from Delta to Cumulative, but not from Cumulative to Delta. When converting from Cumulative to Delta, min and max can be dropped, or captured in an alternative representation such as a gauge.

Reply:
I think we should just calculate min and max when converting from delta to cumulative.

Reply:
Agree, just don't calculate min, max for cumulative -> delta conversion.

Reply:
@srikanthccv 
I've made the changes to only change min/max for cumulative -> delta